### PR TITLE
Update instruction pager for rerouting

### LINKED ISSUE #448: Update current instruction text to signal reroute
Change current instruction text to read "Rerouting" since we no longer announce verbally

### PR DESCRIPTION
**Pull request by sarahlensing**
- Use different adapter to show "Rerouting" in instruction ViewPager

Closes #448

### CODE REVIEW

File: app/src/test/java/com/mapzen/erasermap/view/RouteModeViewTest.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import com.mapzen.erasermap.BuildConfig;
 import com.mapzen.erasermap.PrivateMapsTestRunner;
 import com.mapzen.erasermap.R;
+import com.mapzen.erasermap.adapter.InstructionAdapter;
```
Comment (ecgreb) [inline app/src/test/java/com/mapzen/erasermap/view/RouteModeViewTest.java]: Build is failing to resolve package. Should be `com.mapzen.erasermap.view`.

Comment (sarahlensing) [inline app/src/test/java/com/mapzen/erasermap/view/RouteModeViewTest.java]: Fixed

File: app/src/main/kotlin/com/mapzen/erasermap/view/InstructionAdapter.kt [hunk 2]
```diff
@@ -8,6 +8,8 @@ import android.widget.ImageView
 import android.widget.TextView
 import com.mapzen.erasermap.R
 import com.mapzen.erasermap.util.DisplayHelper
+import com.mapzen.erasermap.view.DistanceView
```
Comment (ecgreb) [inline app/src/main/kotlin/com/mapzen/erasermap/view/InstructionAdapter.kt]: Unused imports

File: app/src/test/java/com/mapzen/erasermap/view/RouteModeViewTest.java [hunk 3]
```diff
@@ -3,6 +3,7 @@
 import com.mapzen.erasermap.BuildConfig;
 import com.mapzen.erasermap.PrivateMapsTestRunner;
 import com.mapzen.erasermap.R;
+import com.mapzen.erasermap.view.InstructionAdapter;
```
Comment (ecgreb) [inline app/src/test/java/com/mapzen/erasermap/view/RouteModeViewTest.java]: Unused import

File: app/src/main/kotlin/com/mapzen/erasermap/view/InstructionAdapter.kt (hunk 2 above)
Comment (sarahlensing) [inline app/src/main/kotlin/com/mapzen/erasermap/view/InstructionAdapter.kt]: Removed

File: app/src/test/java/com/mapzen/erasermap/view/RouteModeViewTest.java (hunk 3 above)
Comment (sarahlensing) [inline app/src/test/java/com/mapzen/erasermap/view/RouteModeViewTest.java]: Removed

**Pull request merged**